pkg/check/kademlia/kademlia.go

```diff
@@ -60,3 +188,32 @@ func Check(cluster *bee.Cluster) (err error) {
 
 	return
 }
+
+func checkKademliaD(topologies bee.ClusterTopologies) (err error) {
```

Comment:
this never will return an error, since the allocation in `L202` will shadow the named return. Under all error conditions, the error will only be printed, but never returned.

Reply:
Same reason as above. Kademlia check needs to be redefined.